openverse_catalog/docs/data_models.md

```diff
@@ -0,0 +1,57 @@
+TODO: This documentation is temporary and should be replaced by more thorough documentation of our DB fields in https://github.com/WordPress/openverse-catalog/issues/783>
+
+# Data Models
+
+The following is temporary, limited documentation of the columns for each of our Catalog data models.
+
+## Required Fields
+
+| field name | description |
+| --- | --- |
+| *foreign_landing_url* | URL of page where the record lives on the source website. |
+| *audio_url* / *image_url* | Direct link to the media file. Note that the field name differs depending on media type. |
+| *license_info* | LicenseInfo object that has (1) the URL of the license for the record, (2) string representation of the license, (3) version of the license, (4) raw license URL that was by provider, if different from canonical URL |
+
+## Optional Fields
+
+The following fields are optional, but it is highly encouraged to populate as much data as possible:
+
+| field name | description |
+| --- | --- |
+| *foreign_identifier* | Unique identifier for the record on the source site. |
+| *thumbnail_url* | Direct link to a thumbnail-sized version of the record. |
+| *filesize* | Size of the main file in bytes. |
+| *filetype* | The filetype of the main file, eg. 'mp3', 'jpg', etc. |
```

Comment:
Should we add something about the fact that filetype can be extracted from the file URL?
https://github.com/WordPress/openverse-catalog/blob/4a9c008a150b2fc5a88b8f9c64f5bd5beef66ac9/openverse_catalog/dags/common/storage/media.py#L293-L304